skt/kernelbuilder.py

```diff
@@ -357,6 +406,49 @@ def find_tarball(self):
 
         return fpath
 
+    def append_to_tarball(self, tarball, extras):
+        """
+        Append `extras` (a list of filename paths rel. to kernel root) to the
+        tarball, if any. To avoid introducing further dependencies
+
+        Args:
+            tarball: Path to a tarball.
+            extras:  A list of file paths (relative to the kernel tree root) to
+                     append.
+        """
+        if not extras:
+            return
+
+        tarball_tmp = os.path.abspath(tarball)
```

Comment:
Tarball path should always be absolute at this point so this step is not necessary.